10up/ads-txt
# Ads.txt file exists check from backend side

### Description of the Change

This PR fixes the https://github.com/10up/ads-txt/issues/48 issue, by adding ads.txt file check from the backend side


Closes #

### How to test the Change


- Put an ads.txt in the root of a MS instance
- Set up domain mapping for a subsite
- Go to the ads.txt edit screen on the subsite
- See notice that an existing ads.txt file was found

### Changelog Entry

> Added - ads.txt file exists check from backend side


### Credits

Props @sksaju, @mmcachran, @peterwilsoncc, @dinhtungdu, @helen, @jeffpaul. 


### Checklist:
